pkgconfig/libvvdec.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=@CMAKE_INSTALL_BINDIR@
+libdir=@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_LIBDIR@
+includedir=@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_INCLUDEDIR@
+
+Name: libvvdec
+Description: Fraunhofer Versatile Video Decoder (VVdeC)
+Version: @PROJECT_VERSION@
+Libs: -L${libdir} -lvvdec -lstdc++
```

Comment:
I would like to leave a note that technically the user can choose the standard c++ library if they use clang, so this could be wrong, however, at the same time, there is no easy way to fix this

Reply:
Hi @1480c1,
Thanks for the suggstion. 
Could you give more suggstion about how to fix this? add a comment in pkg-config?
thanks

Reply:
As far as I know, there's no real way to solve this unless you are willing to parse the output of `${CXX} -v` and try to find the c++ library